internal/testutils/pythoncoverage.go

```diff
@@ -84,8 +84,8 @@ exec python3-coverage run -a %s $@
 			return
 		}
 
-		// Convert to text format
-		coverDir := filepath.Dir(testGoCoverage)
+		// Convert to text format in a subdirectory named after the python coverage file.
+		coverDir := pythonCoverageFile + ".annotated"
 		// #nosec G204 - we have a const for coverageCmd
 		out, err := exec.Command(coverageCmd, "annotate", "-d", coverDir, "--include", tracedFile).CombinedOutput()
 		require.NoErrorf(t, err, "Teardown: can’t combine python coverage: %v", string(out))
```

Comment:
Weirdly enough, I only have a comment on a line that wasn't changed...
Can't we just do ```require.NoError(t, err, "Teardown: can’t combine python coverage: %s", out)```? It looks more idiomatic, I think.

Reply:
good one. Still using NoErrorf though.